src/uct/tcp/tcp_ep.c

```diff
@@ -1677,7 +1676,7 @@ ucs_status_t uct_tcp_ep_am_short_iov(uct_ep_h uct_ep, uint8_t am_id,
     uct_tcp_am_hdr_t *hdr  = NULL;
     struct iovec *iov      = ucs_alloca((uct_iov_cnt + 1) * sizeof(*iov));
     ucs_iov_iter_t uct_iov_iter;
-    size_t iov_cnt;
+    size_t UCS_V_UNUSED length;
```

Comment:
maybe rename it to "payload_length" for consistency with uct_tcp_ep_am_short() ?